### PR TITLE
intro.js apply

### PR DESCRIPTION
**Pull request by songeunwoo**
Additional description used to enhance the user's accessibility page

@junoyoon @GwonGisoo Please review the code.
thank you.

### CODE REVIEW

File: ngrinder-controller/src/main/resources/messages_cn.properties [hunk 1]
```diff
@@ -423,3 +423,35 @@ operation.announcement.title=\u516c\u544a\u7f16\u8f91\u5668
 operation.announcement.help=\u53ef\u4ee5\u4f7f\u7528HTML\uff0c\u6216\u8005\u53ea\u4f7f\u7528\u5236\u8868\u7b26\u63a7\u5236\u6587\u672c\u7f29\u8fdb\u3002
 announcement.title=\u516c\u544a
 
+#intro-js description
+intro.public.button.show=Click to view a description.
+
+intro.index.quick.start=This is a place that can fast start.
```
Comment (junoyoon) [inline ngrinder-controller/src/main/resources/messages_cn.properties]: This is the url field for quick start.

File: ngrinder-controller/src/main/resources/messages_cn.properties [hunk 2]
```diff
@@ -423,3 +423,35 @@ operation.announcement.title=\u516c\u544a\u7f16\u8f91\u5668
...
+intro.index.test.url=Enter the url to the test here.
```
Comment (junoyoon) [inline ngrinder-controller/src/main/resources/messages_cn.properties]: Enter the target URL to be tested here.

File: ngrinder-controller/src/main/resources/messages_cn.properties [hunk 3]
```diff
@@ -423,3 +423,35 @@ operation.announcement.title=\u516c\u544a\u7f16\u8f91\u5668
...
+intro.index.select.language=Select the language of the script can be created.
```
Comment (junoyoon) [inline ngrinder-controller/src/main/resources/messages_cn.properties]: Select the script language which you want to use.

File: ngrinder-controller/src/main/resources/messages_cn.properties [hunk 4]
```diff
@@ -423,3 +423,35 @@ operation.announcement.title=\u516c\u544a\u7f16\u8f91\u5668
...
+intro.index.create=If you want to create a test, press the button.
```
Comment (junoyoon) [inline ngrinder-controller/src/main/resources/messages_cn.properties]: press the create button.

File: ngrinder-controller/src/main/resources/messages_cn.properties [hunk 5]
```diff
@@ -423,3 +423,35 @@ operation.announcement.title=\u516c\u544a\u7f16\u8f91\u5668
...
+intro.index.qna=Q & A bulletin board - can see the questions and answers on how to use ngrinder here.
+intro.index.resource=Resources bulletin - can get a ngrinder resources.
+
+intro.list.create=The click of a button to create a text file.
+intro.list.delete=If you want to delete the selected text, press the button.
+intro.list.search=Tags, keywords, it is possible to search in accordance with the state value.
+intro.list.perftest.status=It indicates the status of the test. Try over-the mouse For more information, see.
+intro.list.perftest.actions=Stop the graph to view the detailed work, you can delete process.
+
+intro.detail.testName=You can enter the name of the test.
```
Comment (junoyoon) [inline ngrinder-controller/src/main/resources/messages_cn.properties]: You can enter the test name here.

File: ngrinder-controller/src/main/resources/messages_cn.properties [hunk 6]
```diff
@@ -423,3 +423,35 @@ operation.announcement.title=\u516c\u544a\u7f16\u8f91\u5668
...
+intro.detail.tags=You can enter the tag of the test.
```
Comment (junoyoon) [inline ngrinder-controller/src/main/resources/messages_cn.properties]: You can put the tag on the test.

File: ngrinder-controller/src/main/resources/messages_en.properties [hunk 7]
```diff
@@ -429,4 +429,35 @@ operation.announcement.title=Announcement Editor
...
+intro.config.basic.script=ou can select the script source.
+intro.config.basic.scriptResources=The resource script file is displayed.
+intro.config.basic.target=The test server will be displayed.&lt;br&gt;For the test, press the Add button to add the destination server.
+intro.config.basic.duration=You can set the duration to be tested.
+intro.config.basic.runcount=You can set the number of threads per run.
+intro.config.rampup=It can be applied to the ramp test.
+
+intro.report.summary=Test results indicate the summary.
+intro.report.tpsGraph=TPS graph shows the information, when you want more detailed information, please click the 'detailed report' button.
+intro.report.testComment=You can enter a comment in the test.
```
Comment (junoyoon) [inline ngrinder-controller/src/main/resources/messages_en.properties]: Please review the english.

File: ngrinder-controller/src/main/resources/messages_cn.properties (hunk 1 above)
Comment (songeunwoo) [inline ngrinder-controller/src/main/resources/messages_cn.properties]: Modified and completed.

**Pull request merged**